saleor/extensions/plugins/webhook/tasks.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+import requests
+from celery import shared_task
+from requests.exceptions import RequestException
+
+from ....webhook import WebhookEventType
+from ....webhook.models import Webhook
+from . import create_webhook_headers
+
+logger = logging.getLogger(__name__)
+
+WEBHOOK_TIMEOUT = 10
+
+
+@shared_task
```

Comment:
I think we should use `saleor.celeryconf.app.task` here. Look for other Celery tasks, where it is used. BTW we should change Avatax task similarly.

Reply:
I am not sure of that. I wanted to limit the dependencies of Saleor in Plugin code. This could be helpful in case of extracting Plugin to different repositories.

Reply:
Done